mongodb/vibe/db/mongo/connection.d

```diff
@@ -627,3 +680,51 @@ private int sendLength(ARGS...)(ARGS args)
 	else if (ARGS.length == 0) return 0;
 	else return sendLength(args[0 .. $/2]) + sendLength(args[$/2 .. $]);
 }
+
+struct ServerDescription
+{
+	enum ServerType
+	{
+		unknown,
+		standalone,
+		mongos,
+		possiblePrimary,
+		RSPrimary,
+		RSSecondary,
+		RSArbiter,
+		RSOther,
+		RSGhost
+	}
+
+@optional:
+	string address;
+	string error;
+	// we are supposed to (MUST) log this value but I can't find what type this is and my server doesn't send it to me anyway...
+	//??? roundTripTime;
```

Comment:
It's a float in Ruby: http://api.mongodb.com/ruby/current/Mongo/Server/Description.html#average_round_trip_time-instance_method